pkg/devfile/parser/parse.go

```diff
@@ -46,6 +48,61 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	DevfileName = "devfile.yaml"
```

Comment:
You can make this package scoped if it's not used anywhere else i.e. `devfileName` instead of `DevfileName`

Reply:
Actually, you can delete this and use the existing `OutputDevfileYamlPath` constant instead